src/Parser/YaccParser.mly

```diff
@@ -153,26 +230,132 @@ expr
 | KW_EFFECT expr_250_list effect_resumption_opt ARROW2 expr
     { make (EEffect($2, $3, $5)) }
 | KW_HANDLER expr { make (EHandler $2) }
+| expr_0 { $1 }
+;
+
+expr_no_comma
+: def_list1 KW_IN expr_no_comma  { make (EDefs($1, $3)) }
+| KW_FN expr_250_list1 ARROW2 expr_no_comma { make (EFn($2, $4))   }
+| KW_EFFECT expr_250_list effect_resumption_opt ARROW2 expr_no_comma
+    { make (EEffect($2, $3, $5)) }
+| KW_HANDLER expr_no_comma { make (EHandler $2) }
+| expr_0_no_comma { $1 }
+;
+
+
+//Evaluate first expresion then discard the resulte and after that evaluate second expression
```

Comment:
What is the role of this comment?

Reply:
It was early comment that I should have deleted. It was only for me to think about operators and what they do.